x-pack/plugins/security/server/deprecations/kibana_dashboard_only_role.ts

```diff
@@ -0,0 +1,244 @@
+/*
```

Comment:
__note:__ I don't try to analyze Advanced Settings to figure out if users change the default role for the Dashboard only mode as it turned out to be a non-trivial task: as far as I'm aware we cannot use UI Settings client to retrieve settings from all spaces, and at the same time we cannot use SO Client directly since we need to account for UI Settings overrides coming from predefined defaults and `kibana.yml`. It feels like the amount of complexity wouldn't be justified for this deprecation.